src/lib/gundb/UserPropertiesClass.js

```diff
@@ -133,7 +131,15 @@ export default class UserProperties {
     const { data, propsNode } = this
 
     assign(data, properties)
-    await propsNode.secretAck(data)
+
+    try {
+      await propsNode.secretAck(data)
+    } catch ({ err: message }) {
```

Comment:
Change to 
catch(e) {
log.error('some text', e.message, e, { some data })
Throw e